### PR TITLE
Add information about contributing custom icons

### PR DESCRIPTION
**Pull request by mcarrano**
This PR exposes the newly documented icon creation process in the Contribute section of the website here: https://www.patternfly.org/v4/contribute/design. This includes a brief introduction and link to the detailed process that exists in the patternfly-design repo.

### CODE REVIEW

**Comment by mcarrano** [comment]
@mmenestr I added a new section at the end of the [design contributors page](https://patternfly-org-pr-2890-v4.surge.sh/v4/contribute/design/) to introduce and link to the process for creating custom icons. Can you give it a review/ Thanks.

File: packages/v4/src/content/contribute/design/design.md [hunk 1]
```diff
@@ -31,4 +31,12 @@ __Example__
 
 To open an issue or browse existing issues, visit the [patternfly-design-kit repo](https://github.com/patternfly/patternfly-design-kit/) on GitHub.
 
+### Contributing new icons
+We encourage designers to work with the existing PatternFly icon set when choosing icons to use in their project. These include both custom and existing Font Awesome icons that cover most common use cases. To see what's available, vist our [Icons page](https://www.patternfly.org/v4/guidelines/icons). There may be occasions where you want to propose a new icon to be added to PatternFly.
```
Comment (mmenestr) [inline packages/v4/src/content/contribute/design/design.md:35]: This reads a little weird. I would maybe add a "However," before "there may be occasions.."

File: packages/v4/src/content/contribute/design/design.md [hunk 2]
```diff
@@ -31,4 +31,12 @@ __Example__
...
+
+__Example__
+*I want to add a new custom icon to PatternFly to meet my project specific need.*
+
+Start by opening an issue in the [patternfly-design](https://github.com/patternfly/patternfly-design/issues) repo proposing a new icon and why it's needed. To contribute a new icon, you can follow the procedure outlined here for [creating custom icons](https://github.com/patternfly/patternfly-design/wiki/Creating-custom-icons) and adding them to the PatternFly icon set.
```
Comment (mmenestr) [inline packages/v4/src/content/contribute/design/design.md:40]: Would just reword this to say:

"To contribute a new icon, start by opening an issue in the [patternfly-design](https://github.com/patternfly/patternfly-design/issues) repo outlining your new icon proposal and why it's needed. Then, follow the procedure outlined here for creating custom icons and adding them to the PatternFly icon set."

Lmk if you don't agree haha

Comment (mcarrano) [reply packages/v4/src/content/contribute/design/design.md:40]: Good suggestion

**Review by mmenestr — APPROVED** [review]
Looks good Matt, thank you!!
